### PR TITLE
fix(ps): parseTime centiseconds multiplier is 10

### PR DESCRIPTION
**Pull request by soyuka**
https://github.com/soyuka/pidusage/issues/69

thanks @jrgm @chqy24

### CODE REVIEW

**Comment by simonepri** [comment]
We need to fix the tests as well

File: lib/ps.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function parseTime (timestr, fraction) {
   var tpart = timestr.split(/-|:|\./)
   var i = tpart.length - 1
   if (i >= 0 && fraction && PLATFORM === 'darwin') { // Fractions of second
```
Comment (simonepri) [inline lib/ps.js]: `Centiseconds` instead of `Fractions of second`

**Comment by simonepri** [comment]
LGM

**Comment by soyuka** [comment]
thanks @simonepri !

**Pull request merged**